recipes/glog/all/patches/fix-cmake.patch

```diff
@@ -1,6 +1,16 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 808330e..9b0f2c4 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -489,7 +489,7 @@ if (WIN32 AND HAVE_SNPRINTF)
+@@ -469,7 +469,6 @@ add_library (glog
+ )
+ add_library(glog::glog ALIAS glog)
```

Comment:
Is this alias needed?

Reply:
It doesn't matter, it's part of patch context. Always tedious to review modifications of patch files 😛

Reply:
ouch! I got confused with those `+`, I thought we were adding those lines here 🤦